functionnal_tests/cypress/integration/app_workspace_advanced/operation_workspace_advanced.js

```diff
@@ -16,7 +16,15 @@ describe('App Workspace Advanced', function () {
   beforeEach(function () {
     cy.loginAs('administrators')
     cy.visitPage({ pageName: p.DASHBOARD, params: { workspaceId } })
-    cy.contains('.userstatus__role__text', 'Space manager')
+
+    // NOTE - MP - 05-11-2021 - If the page isn't loaded after 30s
```

Comment:
Did you add this line to avoid a specific problem? If so could you add it to your comment so that readers will immediately understand the precise situation the timeout is useful in?

Reply:
I'm not sure I'm able to add any more information, @Haskaris could you add some information about this in the next pr you do?

Reply:
I ended up deleting this bit because we found another solution